docs/examples/garden_light.py

```diff
@@ -0,0 +1,11 @@
+from gpiozero import LED, MotionSensor, LightSensor
+from gpiozero.tools import booleanized, all_values
+from signal import pause
+
+garden = LED(2)
+motion = MotionSensor(4)
+light = LightSensor(5)
+
+garden.source = all_values(booleanized(light, 0, 0.1), motion)
```

Comment:
Wouldn't it be better to use `negated(light)` rather than `booleanized(light, 0, 0.1)` ?